.gitlab/scripts/collect_ctest_metrics.sh

```diff
@@ -27,8 +28,9 @@ for i in $(seq 1 ${num_tests}); do
     json_add_value_json "${result_file}" "ctest.testcase" "${ctest_object}"
 
     json_merge "${metadata_file}" "${result_file}" "${result_file}"
-    submit_logstash "${result_file}"
+    cat "${result_file}" >>"${result_file_all}"
```

Comment:
This is the default formatting of `shfmt`. I think it may be possible to change with the `-sr` flag or `space_redirects` editorconfig option (https://github.com/patrickvane/shfmt). If you think it's worth changing it, could you open a PR and/or issue separately?